src/fiatconnect/index.ts

```diff
@@ -27,3 +28,20 @@ export async function getFiatConnectProviders(): Promise<FiatConnectClientConfig
   const { providers } = await response.json()
   return providers
 }
+
+export async function addNewFiatAccount(
+  providerURL: string,
+  fiatAccountSchema: string,
+  properties: any
+): Promise<AddFiatAccountResponse> {
+  const response = await fetch(`${providerURL}/accounts/${fiatAccountSchema}`, {
+    // todo: add auth?
+    method: 'POST',
+    headers: {
+      'content-Type': 'application/json',
+    },
+    body: JSON.stringify({ ...properties }),
+  })
+
+  return response.json()
```

Comment:
This is going to use the SDK, so I think for now its appropriate to just throw a 'Not Implemented' error for this whole function